external/ropsten/UniswapV2Router.json

```diff
@@ -0,0 +1,90 @@
+{
+  // FIXME: This is just a sample file. Replace it with correct values for given network.
+  "address": "0xZZabcd0000000000000000000000000000000001",
```

Comment:
Do we know the address of `UniswapV2Router` contract we want to use for ropsten?

Reply:
I think this is Ropsten's [address](https://ropsten.etherscan.io/address/0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D#code)

Reply:
Thanks, I updated it in 4645d5e63fe5f106cc2c945cb1a20834f72e5e0f